search-api/config/deploy.rb

```diff
@@ -1,15 +1,10 @@
-# rubocop:disable Metrics/BlockLength, Naming/HeredocDelimiterNaming
-
 set :application, "search-api"
 set :capfile_dir, File.expand_path('../', File.dirname(__FILE__))
 set :server_class, "search"
 
 load 'defaults'
 load 'ruby'
 
-set :whenever_command, "govuk_setenv search-api bundle exec whenever"
-require "whenever/capistrano"
```

Comment:
Do you think we'd want to keep this, or will we just add it in next time we want a cron job on search machines?

Reply:
I think we should remove it if we're not using it